From 591834baef44e8222cd073fbb2d55e38a1c65f7d Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 16 Jan 2024 13:39:52 +0100
Subject: [PATCH] Update Makefile

---
 src/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 1047b73..e9878f3 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -55,7 +55,7 @@ endif
 ifndef FORTRAN
   # GNU Fortran as the default Fortran compiler
   # NOTE: use gfortran version 9 or greater, or any compiler compatible with Fortran 2008.
-  FORTRAN=gfortran
+  FORTRAN=$(FC)
   
   # Comment the following line to disable parallelization with OpenMP
   OPENMP=-DOPENMP -fopenmp
@@ -64,7 +64,7 @@ ifndef FORTRAN
   # -fno-backslash prevents backslash characters to be misunderstood
   # -I is used to specify folders where the compiler should look for *.mod files
   # -J is used to specify the folder where the compiler will store the *.mod files
-  FFLAGS=-O2 $(OPENMP) -fno-backslash -I..$(SEP)$(OBJ) -J..$(SEP)$(OBJ)
+  FFLAGS=$(FORTRANFLAGS) -O2 $(OPENMP) -fno-backslash -I..$(SEP)$(OBJ) -J..$(SEP)$(OBJ)
   
   # Add -fpic to compile Atomsk as a shared library
   #FFLAGS+=-fPIC
@@ -87,7 +87,7 @@ endif
 # (for UNIX/Linux systems only)
 # (do NOT insert a / at the end of the paths)
 ifndef INSTPATH
-  INSTPATH=/usr/local
+  INSTPATH=$(PREFIX)
 endif
 # BINPATH: the binary will be copied there
 ifndef BINPATH
